java/server/src/org/openqa/grid/internal/DefaultGridRegistry.java

```diff
@@ -304,7 +304,7 @@ public void add(RemoteProxy proxy) {
     if (proxy == null) {
       return;
     }
-    LOG.info("Registered a node " + proxy);
+    LOG.finest("Registered a node " + proxy);
```

Comment:
This is wildly unhelpful to users --- they need to know when a proxy has been registered.